lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -303,122 +303,45 @@ export class UserAgentApplication {
     }
   }
 
-  //#endregion
+  private successHandler(interactionType: InteractionType, response: AuthResponse, resolve?: any) : void {
```

Comment:
should this be named something else ? success and error handler is a little vague, maybe something that represents the flows?

`tokenSuccessHandler` or something ?

Reply:
responseHandler and authErrorHandler? Since those are the objects we are sending back?

Reply:
willing to punt, as we refactor if we think there are better names can solve then